tests/Articles.js

```diff
@@ -1,112 +1,29 @@
 import Menu from "../page-objects/components/Menu";
-import AllArticles from "../page-objects/pages/AllArticles";
+import Homepage from "../page-objects/pages/Homepage";
 import HomePage from "../page-objects/pages/Homepage";
 
 describe("All the articles", () => {
```

Comment:
Please update a test description with a suggestive one( I see that it is checking on a specific page)